backend/models/users.py

```
@@ -0,0 +1,10 @@
+from sqlalchemy import Column, Integer, String
+from ..database import Base
+
+
+class User(Base):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)